implicit/bpr.pyx

```diff
@@ -221,8 +223,10 @@ class BayesianPersonalizedRanking(MatrixFactorizationBase):
                                                                self.verify_negative_samples)
                 progress.update(1)
                 total = len(user_items.data)
-                progress.set_postfix({"correct": "%.2f%%" % (100.0 * correct / (total - skipped)),
-                                      "skipped": "%.2f%%" % (100.0 * skipped / total)})
+                if total!=0 and total!=skipped:
```

Comment:
nit: Should have spaces around the '!=' operator here for pep8

Reply:
Fixed in the proper PR